src/soplex.h

```diff
@@ -663,6 +663,20 @@ class SoPlexBase
    bool getPrimalReal(R* p_vector, int size);      // For SCIP compatibility
    bool getPrimalRational(VectorRational& vector);
 
+   /// Get the value for the \p i 'th row given the current primal solution
+   bool getRowPrimalValue(int i, R& value);
+   bool getRowPrimalValueRational(int i, Rational& value);
+
+   /// Get the value for the row with indices in \p indices given the current primal solution
+   bool getRowsPrimalValue(const std::set<int>& indices, VectorBase<R>& vector);
+   bool getRowsPrimalValueReal(const std::set<int>& indices, R* p_vector, int i);
```

Comment:
rename the i to something descriptive like size

Reply:
I guess I will use `dim` for consistency

https://github.com/scipopt/soplex/blob/07911134f4c7e478b9131895eeabc15d652ecb29/src/soplex.h#L668